### PR TITLE
Webdriver Custom Property Test Updates

### PR DESCRIPTION
**Pull request by amichaelparker**
### Summary
Ref: [#305](https://github.com/cerner/terra-framework/issues/305). Webdriver tests updated to use themeCombinationOfCustomProperties where possible.

Closes [#305](https://github.com/cerner/terra-framework/issues/305)

### Additional Details
Application utility required having the property combinations divided up for coherence. Other components may require the same after review.

### CODE REVIEW

File: packages/terra-application-layout/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog
```
Comment (StephenEsser) [inline packages/terra-application-layout/CHANGELOG.md]: There is an [open issue](https://github.com/cerner/terra-framework/issues/323) for restructuring these theme tests into a different file. An example of what those changes look like can be found here: https://github.com/cerner/terra-framework/pull/324

Do we want to migrate these theme tests into a new file with this pull request or as a separate issue?

Comment (bjankord) [reply packages/terra-application-layout/CHANGELOG.md]: If you want to tackle https://github.com/cerner/terra-framework/issues/323 in this PR, that sounds good to me. We can scope it to just the packages you've already modified in this PR or do the whole repo. Either way sounds good to me.

+1 on changes so far.

Comment (StephenEsser) [reply packages/terra-application-layout/CHANGELOG.md]: Breaking the tests out is no longer required for the theme repositories to skip these tests. However, there is some discussions around whether or not we want to continue breaking the tests out to segregate functional tests from theme tests. 

cc: @mjhenkes @bjankord

Comment (bjankord) [reply packages/terra-application-layout/CHANGELOG.md]: I propose we check in what's in the PR. It looks like only 2 of the theme tests had functional steps that would be duplicated with the primary wdio tests. The rest looked to be just defining the theme variables statically without any functionality being invoked to verifiy component is in a set state before capturing a screenshot.

With that being said, I wouldn't worry about doing this work in the other repos.

Comment (bjankord) [reply packages/terra-application-layout/CHANGELOG.md]: @amichaelparker  Let's create a branch off of this to keep the changes to split the theming tests into new files.
And then back on this branch, lets revert those change and just keep the code that updates to use themeCombinationOfCustomProperties where possible.

**Pull request merged**